kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/HybridLearningFooter.vue

```diff
@@ -149,7 +149,15 @@
         return !this.isTopic && this.allowDownloads;
       },
       downloadedByLearner() {
-        return this.downloadEnabled && Boolean(this.downloadRequestMap[this.contentNode.id]);
```

Comment:
I don't understand why this is preferable to keying the downloadRequestMap by contentnode_id?

We should revert this change, and instead ensure that the downloadRequestMap is always keyed by contentnode_id.